jetbrains_plugin/src/main/kotlin/com/sina/weibo/agent/extensions/plugin/costrict/CostrictCodeActionConstants.kt

```diff
@@ -0,0 +1,252 @@
+// SPDX-FileCopyrightText: 2025 Weibo, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package com.sina.weibo.agent.extensions.plugin.costrict
+
+/** Type alias for prompt type identifiers */
+typealias CostrictCodeSupportPromptType = String
+/** Type alias for prompt parameters map */
+typealias CostrictCodePromptParams = Map<String, Any?>
+
+/**
+ * Data class representing a prompt configuration with a template string.
+ * Templates contain placeholders that will be replaced with actual values.
+ */
+data class CostrictCodeSupportPromptConfig(val template: String)
+
+/**
+ * Collection of predefined prompt configurations for different use cases.
+ * Each configuration contains a template with placeholders for dynamic content.
+ * 
+ * now organized under the Costrict extension.
+ */
+object CostrictCodeSupportPromptConfigs {
+    /**
+     * Template for enhancing user prompts.
+     * Instructs the AI to generate an improved version of the user's input.
+     */
+    val ENHANCE = CostrictCodeSupportPromptConfig(
+        """Generate an enhanced version of this prompt (reply with only the enhanced prompt - no conversation, explanations, lead-in, bullet points, placeholders, or surrounding quotes):
+
+${'$'}{userInput}"""
+    )
+
+    /**
```

Comment:
在CostrictCodeActionConstants.kt文件中，有一些TODO注释和未完成的实现，需要进一步完善。

```suggestion
// 移除空的TODO注释
// TODO: Add more prompt types as needed

// 在注释中明确说明这个文件是Costrict扩展的一部分
/**
 * Collection of predefined prompt configurations for different use cases.
 * Each configuration contains a template with placeholders for dynamic content.
 * 
 * These configurations are now organized under the Costrict extension.
 */